WordPress/Classes/ViewRelated/System/FilterTabBar.swift

```diff
@@ -356,7 +356,19 @@ class FilterTabBar: UIControl {
         }
     }
 
+    var currentlyCelectedItem: FilterTabBarItem? {
+        guard !items.isEmpty, selectedIndex < items.count else {
```

Comment:
I think you can check if the indices contain this index here:

```suggestion
        guard items.indices.contains(selectedIndex) else {
```

Reply:
Sorry, I just realized there _is_ an extension on Collection for safe access:

https://github.com/wordpress-mobile/WordPress-iOS/blob/c1446d7a1f84812815ace1b8d8ccf9fad15d2e13/WordPress/Classes/Extensions/CollectionType%2BHelpers.swift#L15

Should be something like

```swift
return items[safe: selectedIndex]
```

Reply:
This looks pretty cool!! Thanks for the hint